assets/assets-v2.json

```diff
@@ -0,0 +1,29 @@
+      "dark": "iVBORw0KGgoAAAANSUhEUgAAAEIAAABCCAQAAABJXchjAAACt0lEQVR42s3ZvU8TYRzA8V/aAV/AxUEDLSiGKBKJiRFn5cWJEOJQmU38E9wMpAMUB0NNBEuiAzEgmwmDg39CocRJCBahLcZESliMXNvwdbihvJSW57nn7vjc/ss3uT69u+cRdYQZIso8y6TZwcJihzQpPhHlKWFxE0H6mGaNWtZI0EtATKOZcXKoyBGjWUyhlTh76Cgwwy1xiovEKOJEkTiXRB9P2MKEHIOigzrimDTDeVFDC8uYllJavrSTwQ2/6JTT4QF53JKnS2qjgzxu2qWz9m8hRyUldjSuEpVkCFdfESkqWxUNrFLZN87JSZgCTyLgzcl/TXgWAQNyHA3kPI3IUi9HMQGeRsArOYybFD2PKHBDDuIDqERwlVZqX+tUNy1lhLEUI75gQoGW8shx8CUCxsRGgKxvETmC9sA+8C0Cuu2B075GTNkDf/gasWKvDHyNgJAw5HtERIgajSjxmceoGRHmjUXsEue6CEHUzAopIxHLPLNf6TUiFoUNhxElFugRm17EupB3EPG7/NHrIOKPYGlGLPLcvgGOI/Z0IxrFZihiWy2iNjrUb8eGuQgC9LDAPmrSwpKZCC7zgp/oSArzziO4S4K/6JoVok4iCNLPV5wZFoZ0I7jCS7ZwLqL7KL/HRyzMaBIR1nx9lH+3ByZ8jZi0B/b6GvHwbLzy24j5FjF6Fj4Dm6WM975EvJODaKPoeUSBVjmM155HxFS3i8xHZKiX4xj0MGKffqmMt55FTFTbTF3yJCJJXfWTvuyptpUL6NskJNVxm23ctE271EYXeRcT7p/+0GUTN2xxR+0UeBHTkoR0DuL2MSdBnehggCwmZOgXfVxgBAsnCsRpEKe4Rpx/6LCYoU1MIcwYGVRkGCUkphGgmylWqWWFSR4REDfRRIQR5kiSJo+FRZ40SeYYJkKj+sT/4VHvuFGgLiwAAAAASUVORK5CYII=",
+      "light": "iVBORw0KGgoAAAANSUhEUgAAAEIAAABCCAQAAABJXchjAAACrklEQVR42s3ZT2scZQDH8Q+bwxptvXhQ2kyikaKxWASxnrV/PIVSeog5C74Eb9KQQ7v1IF3BxgT0EKQ1N6EHD76ETRM8aYhb281GBLOhF7G7CXk8B7NmZ+bZGb+/+/KBgZ2ZZ2QoMWveinVNu7q6djWt+c68DyWG2ojLlmwKx2zToksqojfulraQYm0146I1qe6pkGE9y16Xu+fU7Ak5tqfueTm6ZluIsLarMlVVFyJu2aiUTVgXIm9NIkVTWoL4+905A/aujjCkdZw3QGd1hCHuiXOOaaLPX9K+3QzbF45YS0L/qtb6+DdkaUM4cj95Rt8WhEIQwRf6dE0oDBFccUQntQtFbDnhX90WCkUEn3G41+wVjuh51aG+EVIhXjI5wB4K/7klAEh0UyJ+ECKsZwKAW0IpiOAmQMVWaYi2EeCyUBoiuAAslYpYAH4tFfELJEKpiGCM2dIRM8xHRez73gcpEXOsREM8UfcKRlIi7rIWBbHuI6OQAbHKo5yIffddBDIiHtLJgfhDzTjkQvxJNyNi1cdGITfiaVbEKSAWYicFYrDOpr8cjyIiKi667yAlosmDSIgXfOI3IcMarERAvGXRX0LG3WU+F2LEtB+FXLvObGbEiz61HecGlmRCvO1bXSHKTsNmqbfyn4HFUhF3gEulIt77Hz3yUysNcYPyXwPHAeDrUhBfOdQZe4UjeiY53OeFI2ppj4viI1pOOKKrBSIOTOvTl4Uhbutb1YNCEA1V+pfYGuhYuZeD8NiYY3rDjjDE7ZgyQOd1hkh4x4BNeTwUwrY3pSixGp3QMJblQ9xBRMKiqkxdsRUF0DItR8+a080F6Kk7KXcvq/s7E6Br2RnRStzUSnkJbhgTvYoLFmwMcDJ5x/sqhtppM+bc09DU0dXV0dRwz3UzTqX/wX8AL5Yi61hKoK8AAAAASUVORK5CYII="
+    }
+  }
```

Comment:
Why do we have to do a v2?

Reply:
Because the structure changed for the naming. We cannot break old SDKs that might be in use.

If we were only adding, not renaming, it would be fine.